pkg/executor/client/job.go

```diff
@@ -225,6 +230,9 @@ func (c *JobExecutor) Execute(ctx context.Context, execution *testkube.Execution
 	if err != nil {
 		return result.Err(err), err
 	}
+
+	c.logsStream.Push(ctx, execution.Id, events.NewLog("created kubernetes job"))
```

Comment:
shoud it be for v2 only?

Reply:
👍🏻

Reply:
yeah need to pass feature flags too :/ good point!